ani-cli

```diff
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/usr/bin/env sh
```

Comment:
This breaks the script on a normal arch installation

Reply:
Similar problems might be present on termux